lib/pure/uri.nim

```diff
@@ -56,6 +85,10 @@ proc encodeUrl*(s: string, usePlus=true): string =
   ##
   ## As a special rule, when the value of ``usePlus`` is true,
   ## spaces are encoded as ``'+'`` instead of ``'%20'``.
```

Comment:
General remark: I would love to see "See also:" section which will give links to similar procs. Some examples: [strutils.indent](https://nim-lang.github.io/Nim/strutils.html#indent%2Cstring%2CNatural%2Cstring), [tables.getOrDefault](https://nim-lang.github.io/Nim/tables.html#getOrDefault%2CTable[A%2CB]%2CA), etc.

Here, specifically, a link to `decodeUrl` would be useful. (And vice-versa)

See if you can do the same for some other procs too.

Reply:
Ok? ba767c59235c0f385326f306660f32ce1750cddb